### ISSUE TITLE
what's the difference between processes and guides?

### ISSUE THREAD

**Issue by ahdinosaur** [enspiral-root-systems/meta#64]

**Comment by danalexilewis** [comment]
Good question. For me processes are things we have agreed on - where as a guide is just a brain dump. Processes require the lowest level descision guides don't.

**Comment by danalexilewis** [comment]
At least that was the initial idea

**Comment by ahdinosaur** [comment]
thanks for the response. so then, what's the difference between processes and agreements?

**Comment by danalexilewis** [comment]
Yeah I was thinking that as I wrote it. Probably it's a type of agreement. Though our financial and financial agreement could be said to be a process  but the diversity one is a value agreement. I'm open to collapsing them just into agreements
